config-example.yml

```diff
@@ -6,30 +6,18 @@ graphhopper:
   graph.location: graph-cache
 
 
-  ##### Vehicles #####
-
-  # More options: foot,hike,bike,bike2,mtb,racingbike,motorcycle,wheelchair (comma separated)
-  # bike2 takes elevation data into account (like up-hill is slower than down-hill) and requires enabling graph.elevation.provider below.
-  # graph.flag_encoders: car
-
-  # Enable turn restrictions for car or motorcycle.
-  # graph.flag_encoders: car|turn_costs=true
-
-  # Add additional information to every edge. Used for path details (#1548), better instructions (#1844) and tunnel/bridge interpolation (#798).
-  # Default values are: road_class,road_class_link,road_environment,max_speed,road_access (since #1805)
-  # More are: surface,smoothness,max_width,max_height,max_weight,hgv,max_axle_load,max_length,hazmat,hazmat_tunnel,hazmat_water,toll,track_type,
-  #           mtb_rating, hike_rating,horse_rating,lanes
-  # graph.encoded_values: surface,toll,track_type
-
   ##### Routing Profiles ####
 
-  # Routing can be done for the following list of profiles. Note that it is required to specify all the profiles you
-  # would like to use here. The fields of each profile are as follows:
+  # Routing can be done only for profiles listed below. A car profile is enabled per default and can be changed with
+  # tweaking the custom_model. For more information about profiles and custom profiles have a look into the
```

Comment:
This sounds like there is some special default car profile that is always enabled besides the ones in the list. I would just leave out the sentence "A car profile...". What is enabled and what is not can easily be seen by looking at the list.